PXWeb/Resources/Styles/main-pxweb.css

```diff
@@ -3230,7 +3230,8 @@ span.AspNet-TreeView-Collapse::before {
   font-size: smaller; }
 
 #pxwebcontent .variableselector_valuesselect_hiddeneventbutton_panel {
-  height: 22px; }
+  /*    height: 22px;*/
+  display: none; }
```

Comment:
Maybe the panel-tag with id=HiddenEventButtons  should be removed from VariableSelectorValueSelect.ascx. Is it in use anymore?